Tests/SpeechUnitTests.cs

```diff
@@ -279,7 +279,7 @@ public void TestSectorTranslations()
         [TestMethod]
         public void TestSpeechHumanize1()
         {
-            Assert.AreEqual("well over minus 12 thousand", Translations.Humanize(-12345));
+            Assert.AreEqual("well over negative 12 thousand", Translations.Humanize(-12345));
```

Comment:
Maybe this is a UK/US thing, but over here we would say "minus", at least in everyday speech.

Reply:
I was thinking of negative as the opposite of positive, but upon reflection and after reviewing norms for describing temperatures both in the US and UK I have to agree. I've reverted from "negative" to "minus". o7